ampd/src/broadcaster.rs

```diff
@@ -48,15 +49,20 @@ pub enum Error {
     ExecutionError { response: TxResponse },
 }
 
-#[derive(Debug, Deserialize, Clone)]
+#[serde_as]
+#[derive(Debug, Deserialize, Serialize, Clone, PartialEq)]
 pub struct Config {
     pub chain_id: Id,
+    #[serde(rename = "tx_fetch_interval_in_ms")]
+    #[serde_as(as = "serde_with::DurationMilliSeconds<u64>")]
     pub tx_fetch_interval: Duration,
     pub tx_fetch_max_retries: u32,
     pub gas_adjustment: f64,
     pub gas_price: DecCoin,
     pub batch_gas_limit: Gas,
     pub queue_cap: usize,
+    #[serde(rename = "broadcast_interval_in_ms")]
+    #[serde_as(as = "serde_with::DurationMilliSeconds<u64>")]
```

Comment:
why not use`#[serde(with = "humantime_serde")]`? should be more readable than ms